src/components/v5/shared/StatusText/StatusText.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import clsx from 'clsx';
+import { StatusTextProps } from './types';
+import Icon from '~shared/Icon';
+import { STATUS_TYPES } from './consts';
+
+const displayName = 'v5.shared.StatusText';
+
+const StatusText: FC<StatusTextProps> = ({
+  title,
```

Comment:
I think you can just use children for this

Reply:
Added as children